### DISCUSSION TITLE
New modules

### DISCUSSION THREAD

**Discussion by stickerx** [quer/the-steam-awards#35] [Ideas]
Hi ALL ;)
I am looking for a module 

Follow The Game

Follow CURATORS

so please, if someone is able to do this, I will be very thankful to him

**Comment by SENPAY98K** [comment]
You should know that you need unlimited steam accounts to be able to follow a curator, also feel free to contact me i have over 1100 accounts that can follow your curator :D

**Reply by stickerx** [reply]
1100 unlimited steam account, Gabe If he hears this, he will commit suicide xD

**Reply by SENPAY98K** [reply]
> 1100 unlimited steam account, Gabe If he hears this, he will commit suicide xD

Not really, all services owners are running above 2000 atm

**Comment by quer** [comment]
this have now been added.
https://github.com/quer/the-steam-awards/commit/608555a9716bbf46e3d1bbae2793eae1a91be51f
